### PR TITLE
added

### PR DESCRIPTION
**Pull request by theanimator20**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1367,14 +1367,44 @@ <h2>Help your friends find some good learning resources</h2>
...
-                <!--contributor card end-->
+          <!-- ________Keegan Colaco Contributor card START ________  -->
+            <div class="card">
+                <p class="name">Keegan Colaco</p>
+                <p class="contact">
+                    <i class="fab fa-twitter"></i>
+                    <a href="https://twitter.com/KeeganColaco20" target="_blank">@KeeganColaco20</a>
+                </p>
+                <p class="about">
+                    I'm Keegan Colaco a Computer Engineering student   having keen interest in Frontend Web Development. Always Up for working with cool open source Projects.
```
Comment (Nitesh-thapliyal) [inline index.html:1380]: can you remove the white space in your text

**Review by Nitesh-thapliyal — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (theanimator20) [reply index.html:1380]: Yh i will remove them

**Comment by theanimator20** [comment]
i have solved the issue please Accept The PR

**Comment by Nitesh-thapliyal** [comment]
@theanimator20 can you resolve the conflict

**Comment by theanimator20** [comment]
Resolved

**Comment by Nitesh-thapliyal** [comment]
> Resolved

there is still a conflict in file

**Comment by theanimator20** [comment]
Ok

**Pull request closed without merging**